LinkedIn Ads source documentation

mohan-h opened the pull request:
Initial commit

mohan-h committed:
LinkedIn Ads source documentation

Initial commit

carlojavier commented on help/sources/tutorials/api/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,1060 @@
+| `accountIds` | Your LinkedIn account IDs |
+| `campaignGroupIds` | Your LinkedIn Campaign Group IDs. |
+| `campaignIds` | Your LinkedIn Campaign IDs. |
+| `pivot` | One of `ACCOUNT`, `CAMPAIGN GROUP`, `CAMPAIGN` or `CREATIVE` |
```
Can we add a brief definition or description to what `pivot` is?

carlojavier commented on help/sources/tutorials/api/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,1060 @@
+>
+>The [!DNL LinkedIn Ads] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campa ...
```
- Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
- And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

carlojavier commented on help/sources/tutorials/api/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,1060 @@
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campa ...
+
+This Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) to retrieve your LinkedIn campaign data, along with all performance and metrics into Experience Platform allowing you to execute ana ...
```
- Some restructuring here: "...to retrieve your LinkedIn campaign data, along with all performance and metrics. You can then bring that data to Experience Platform, where you can execute analytics."
- "The data is returned from a specified date with a default of six month retention period for daily data." (Let me know if this rephrasing still works for the meaning.)

mohan-h replied:
Used _The data is returned from a specified date within the six month default retention period for daily data._

mohan-h replied:
Used _The data is returned from a specified date within the six month default retention period for daily data._

carlojavier commented on help/sources/tutorials/api/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,1060 @@
+| Access Token | Assuming you have a LinkedIn Developer Application follow this [!DNL LinkedIn] tutorial [Authorization Code Flow (3-legged OAuth)](https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow) to generate the access token. | `AQV4FClIuE2BbDkQeDatkhMFs................qNAmFmlcr9A` |
+| Account IDs | You can find guidance to obtain the Account ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `508670032` |
+| Campaign Group IDs | Campaign Group IDs can be obtained by navigating to the Campaign Groups page and selecting the desired CIDs. | `611138362` |
+| Campaign IDs | You can find guidance to obtain the Campaign Group ID [here](https://www.linkedin.com/help/lms/answer/a424270). | `1991010101,1992020202` |
```
For the API side, can we wrap the credentials in backticks and write them out in camelcase

carlojavier commented on help/sources/tutorials/api/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,1060 @@
+
+### Guardrails {#guardrails}
+
+See the Retention, Restrictions, Delays and Accuracy sections on the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) page. Note, [!DNL LinkedIn Ads] only allows data retrieval from campaigns that are active. The data is returned from a specified date for the default 6 month retention on a daily basis.
```
Let's move the second sentence into an actual note:

>[!NOTE]
>
>LinkedIn Ads only allows data retrieval from active campaigns.The data is returned from a specified date with a default of six month retention period for daily data."

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+
+### Validation {#validation}
+
+To validate that you have correctly set up the source; and [!DNL LinkedIn Ads] events are being ingested, follow the steps below:
```
Semi-colon (`;`) is unnecessary here

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+To validate that you have correctly set up the source; and [!DNL LinkedIn Ads] events are being ingested, follow the steps below:
+
+1. In the Platform UI, select View Dataflows besides the [!DNL LinkedIn Ads] card menu on the Catalog page. You can then select [!UIControl Preview dataset] to verify the data that was ingested.
+![Platform UI screenshot for LinkedIn Ads Preview Dataset](../../../../images/tutorials/create/linkedin-ads/preview-dataset.png?lang=en)
```
Can the screenshot be updated here? There is a typo on the dataset name "Tset123"

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+
+## Overview
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campa ...
```
Same as API:
Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+
+### Guardrails {#guardrails}
+
+See the Retention, Restrictions, Delays and Accuracy sections on the [LinkedIn Reporting & ROI APIs](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads-reporting/ads-reporting) page. Note, [!DNL LinkedIn Ads] only allows data retrieval from campaigns that are active. The data is returned from a specified date for the default 6 month retention on a daily basis.
```
Same comment as API about moving the note into its own [!NOTE] tag

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+## Next steps
+
+By following this tutorial, you have established a connection to your [!DNL LinkedIn Ads] account. You can now continue on to the next tutorial and [configure a dataflow to bring data into Platform](/help/sources/ui-tutorials/dataflow/crm.md).
+
```
I think we need to preface this particular part about why a user needs to schedule specifically for Day or Hour? Can we add a brief intro here on why this is recommended/required?

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+
+> [!NOTE]
+>
+> Multiple IDs can be specified by passing comma separated values.
```
Need to remove space between ">" and the text

carlojavier commented on help/sources/tutorials/ui/create/advertising/linkedin-ads.md:
```diff
@@ -0,0 +1,150 @@
+| --- | --- |
+| accountIds | Your LinkedIn account IDs. |
+| campaignGroupIds | Your LinkedIn Campaign Group IDs. |
+| campaignIds | Your LinkedIn Campaign IDs. |
```
let's wrap the entries in the fields column in backticks

carlojavier reviewed (changes requested):
@mohan-h looking good, some comments added!

mohan-h committed:
Merge branch 'AdobeDocs:main' into linkedin-ads-source

mohan-h committed:
Updates post review

Address review comments https://github.com/AdobeDocs/experience-platform.en/pull/104#pullrequestreview-1086083387

mohan-h committed:
Merge branch 'AdobeDocs:main' into linkedin-ads-source

mohan-h committed:
Merge branch 'AdobeDocs:main' into linkedin-ads-source

mohan-h committed:
Corrected wrong attribute reference

carlojavier commented:
Closing due to inactivity. The PR is ready on internal, still just waiting on approval from internal on when it can be published.
